src/icon/avatarIcon.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { PureComponent } from 'react'
+import cc from 'classcat'
+import isEmpty from 'lodash.isempty'
+
+import { color } from '_utils/branding'
+
+class AvatarIcon extends PureComponent<Icon> {
+  static defaultProps: Icon = {
+    className: '',
+    iconColor: color.icon,
+    size: 24,
+    title: '',
+  }
+
+  render() {
+    const { className, iconColor, size, title } = this.props
+    return (
+      <svg
+        viewBox="0 0 19 20"
+        xmlns="http://www.w3.org/2000/svg"
+        className={cc(['kirk-icon', className])}
+        width={size}
+        height={size}
+        aria-hidden={isEmpty(title)}
+        fill={iconColor}
+      >
+        {title && <title>{title}</title>}
+        <g fill="none">
+          <path d="M-2-2h24v24H-2z"/>
```

Comment:
What is this path used for? I noticed you have one on each icon and it seems that it's useless.